index.js

```diff
@@ -318,6 +320,24 @@ class ServerlessCustomDomain {
       throw new Error(`Cannot find specified domain name ${this.givenDomainName}.`);
     });
   }
+
+  /**
+   * Prints out a summary of all domain manager related info
+   */
+  domainSummary() {
+    this.getDomain().then((data) => {
+      this.serverless.cli.consoleLog(chalk.yellow.underline('Serverless Domain Manager Summary'));
+      if (this.serverless.service.custom.customDomain.createRoute53Record !== undefined
```

Comment:
How would the code change (aside from removing the `undefined` check) by treating `createRoute53Record` as if it defaulted to `true`?